From 8d301cdc1729cd21d53fd1b3f30c34775994b3b0 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 23 May 2022 15:24:26 -0600
Subject: [PATCH] Update makefile.gnu

---
 src/makefile.gnu | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makefile.gnu b/src/makefile.gnu
index f7e3062b4..f1da69ee0 100644
--- a/src/makefile.gnu
+++ b/src/makefile.gnu
@@ -12,7 +12,7 @@ PROJECT_EIGEN=/usr/include/eigen3/
 # COMPILERS AND FLAGS
 ###############################################################################
 PROJECT_CC=g++
-PROJECT_MPICC=mpic++
+PROJECT_MPICC=mpicxx
 # OpenMP parallelization is disabled by default, add flag "-fopenmp" to enable.
 PROJECT_CFLAGS=-O3 -march=native -std=c++11
 PROJECT_CFLAGS_MPI=-Wno-long-long
